src/coreComponents/mesh/generators/VTKMeshGenerator.cpp

```diff
@@ -562,7 +674,7 @@ splitCellsByType( vtkDataSet & mesh )
   // Collect cell lists for each type (using array for speed in a hot loop)
   for( vtkIdType c = 0; c < numCells; c++ )
   {
-    ElementType const elemType = convertVtkToGeosxElementType( static_cast< VTKCellType >( mesh.GetCellType( c ) ) );
+    ElementType const elemType = convertVtkToGeosxElementType( static_cast< vtkCell * >( mesh.GetCell( c ) ) );
```

Comment:
I do not think you need to cas `mesh.GetCell( c )` in to a `mesh.GetCell( c )`, do you?